.husky/.gitignore

```diff
@@ -0,0 +1 @@
+_
```

Comment:
what does this do exactly?

Reply:
Yeah, it's auto-generated. The `husky/_` folder seems to be generated during the pre-commit check as a temporary workspace